src/Shared/PackageManagers/NPMProjectManager.cs

```diff
@@ -215,11 +216,14 @@ public override async Task<PackageMetadata> GetPackageMetadata(PackageURL purl)
                     metadata.VersionUri = $"{ENV_NPM_ENDPOINT}/package/{metadata.Name}/v/{metadata.PackageVersion}";
                     metadata.ApiVersionUri = $"{ENV_NPM_API_ENDPOINT}/{metadata.Name}/{metadata.PackageVersion}";
 
+                    // sometimes the description is only on the version level
+                    metadata.Description ??= OssUtilities.GetJSONPropertyStringIfExists(versionElement, "description");
```

Comment:
nit: might it make more sense to prioritize the version specific description over the package one if it exists on the version?